### DISCUSSION TITLE
display Virtual machines and devices in the same table

### DISCUSSION THREAD

**Discussion by nokiaIL** [netbox-community/netbox#17024] [Q&A]
hello, 

is it possible to view virtual machines and devices in the same table? 
thanks,

**Comment by candlerb** [comment]
Not in the existing Netbox GUI, no. They are different entities, and the namespaces overlap (e.g. device id=1 and vm id=1 can both exist).

You could write a plugin.

**Reply by nokiaIL** [reply]
thanks for the answer, i will start looking on the plugin documents.
I'm little bit new for this - do you have materials or kind of reference I can look at to get the idea?

**Reply by candlerb** [reply]
https://github.com/netbox-community/netbox-plugin-tutorial

**Comment by ghost** [comment]
Couldn't you accomplish this with tags?
a virtual machine is not a physical server, and vice versa. So like @candlerb said, the models are completely different. 
BUT, if you were to tag both the servers you want to include, and the VM's with the same tag, then filter/search by the tag I don't see why this wouldn't be possible.
